SingularityService/src/main/java/com/hubspot/singularity/config/UIConfiguration.java

```diff
@@ -59,6 +59,10 @@ public static RootUrlMode parse(String value) {
   @NotNull
   private List<ShellCommandDescriptor> shellCommands = Collections.emptyList();
 
+  @JsonProperty
+  @NotNull
+  private Boolean showTaskDiskSpace = false;
```

Comment:
i know this is a nitpick, but `showTaskDiskResource` better describes what this is